app/src/main/res/values-sw600dp-land/integers.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+  <integer name="topic_description_collapsed">100</integer>
```

Comment:
Remove this integer. This is incorrect approach, as discussed today.

Reply:
Done